src/herder/test/TransactionQueueTests.cpp

```diff
@@ -1037,8 +1037,8 @@ TEST_CASE("replace by fee", "[herder][transactionqueue]")
         {
             for (uint32_t i = 1; i <= 3; ++i)
             {
-                test.add(transaction(*app, account1, i, 1, 199),
-                         TransactionQueue::AddResult::ADD_STATUS_ERROR);
+                auto tx = transaction(*app, account1, i, 1, 199);
```

Comment:
Was this changed for style reasons, or is there something about the semantics that required this?

Reply:
nope, just leftover: I thought I was going to hit `txINSUFFICIENT_FEE` in those cases, but that's not the case. I will remove this when I do the final rebase